### PR TITLE
Add quick-look-plugins reference

### PR DESCRIPTION
**Pull request by unixorn**
Add pointer to a list of extra quicklook plugins.

### CODE REVIEW

**Comment by iCHAIT** [comment]
Please read the [contribution guidelines](https://github.com/iCHAIT/awesome-osx/blob/master/contributing.md) and stick to them.

File: README.md [hunk 1]
```diff
@@ -131,6 +131,7 @@
 - [Path Finder](http://www.cocoatech.com/pathfinder/) - A powerful dual-pane browser alternative to Finder.
 - [TotalFinder](http://totalfinder.binaryage.com/) - A powerful alternative to Finder.
 - [XtraFinder](https://www.trankynam.com/xtrafinder/) - Adds useful features to Finder.
+- [Quicklook-Plugins](https://github.com/sindresorhus/quick-look-plugins) - List of extra quicklook plugins to enable previewing more filetypes in the Finder.
```
Comment (iCHAIT) [inline README.md]: You should place this is in alphabetical order, also it does not has the OSS symbol as mentioned in the contribution guidelines.

**Comment by unixorn** [comment]
@iCHAIT corrected the alphabetization issue. It is a list of plugins, not a specific application, so I didn't think the OSS icon rule applied.

**Comment by iCHAIT** [comment]
Fair enough :+1:

**Pull request merged**